packages/waku/src/router/create-pages.ts

```diff
@@ -588,7 +588,7 @@ export const new_createPages = <
           }
         });
         staticPathMap.set(
-          page.path,
+          '/' + pathItems.join('/'),
```

Comment:
Would you like to add unit tests for this?

Reply:
yea that sounds like a good idea 👀

Reply:
added a test for this, but I need to move `createPages.test.ts` to use `new_createPages` next. I'll make a new PR for that if it's ok